core/source/Locale.mint

```diff
@@ -0,0 +1,19 @@
+module Locale {
+  /* Sets the current locale. */
+  fun set (locale : String) : Bool {
```

Comment:
May be worth making the response to this `Promise(Bool)` in anticipation of loading in languages asynchronously?

Reply:
Yes, good idea, will update it :+1: